static/app/utils/hotkeyslistener.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import {getKeyCode} from './getKeyCode';
+
+const HotkeysListener = ({
```

Comment:
This should probably be a hook not a component right?

Reply:
This was originally made as a component so it could just be slotted into the smart search class component right away. Now that I think of it, this can be made a hook and then just have a functional component to place in the smart search that wraps this